### DISCUSSION TITLE
Connecting to Github repository from Dreamweaver 2021

### DISCUSSION THREAD

**Discussion by AmmarSM** [community/community#31887] [New to GitHub]
Hi
I try to connect to Github repository from Dreamweaver 2021 using user name and password authentication but I get the error message below ? If I understand it clearly this way of connecting to Github is no longer supported ? its the only way on Dreamweaver !

![Screen Shot 2022-09-04 at 2 41 30 AM](https://user-images.githubusercontent.com/1404946/188290856-e4ddd042-c3b5-472e-9117-49d05341f8dd.png)

![Screen Shot 2022-09-04 at 2 45 47 AM](https://user-images.githubusercontent.com/1404946/188290870-8a0dccb9-8d90-4d13-8b0d-14cd4f68277f.png)

**Comment by jfullstackdev** [comment]
did you try Personal Access Token ? 

check this out

https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
